src/configure/helper/graphHelper.ts

```diff
@@ -87,6 +83,14 @@ export class GraphHelper {
 
         return accountName + "-" + projectName + "-" + guid;
     }
+    public static async getAccessToken(session: AzureSession): Promise<string>{
+      const token = await this.getRefreshToken(session);
+      return token.accessToken;
+     }
+
+    private static contributorRoleId = "b24988ac-6180-42a0-ab88-20f7382dd24c";
```

Comment:
data members should come in the beginning of a class.

Reply:
yes, that correct but private data members should come after public data members thats i moved them after public functions